packages/gestalt/src/Table.css

```diff
@@ -19,6 +19,7 @@
   background-color: var(--g-colorGray0);
   position: sticky;
   top: 0;
+  z-index: 1;
```

Comment:
Not 100% sure about this, but should it be using the z-index helpers so we can track this better?

Reply:
could we do something like

`  return props.sticky ? <Sticky zIndex={new FixedZIndex(1)} > <thead className={cs}>{props.children}</thead> </Sticky > : <thead className={cs}>{props.children}</thead>;
`

Reply:
@chrislloyd that's the approach I went with before, to add a `zIndex` prop to `<Table.Header />`. But the zIndex needs to be set on the `th`, not the `thead`. The `th` is also not aware of the `thead` so that's why I ended up with this initial fix (set the zIndex to 1).

See https://css-tricks.com/position-sticky-and-table-headers/
> You can’t position: sticky; a <thead>. Nor a <tr>. But you can sticky a <th>

@AlbertCarreras no, sadly that's not correct html. the `thead` should be an immediate child of the `table`. If we wrap it in `<Sticky />`, that's no longer the case.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/thead

![image](https://user-images.githubusercontent.com/127199/92827047-3f37a500-f386-11ea-9286-6139198e25f2.png)

Reply:
Ah great to see you already tried this. I was just suggesting using it internally might be better than using the CSS var directly but I don't have a strong opinion :)